tools/lou_checkyaml.c

```diff
@@ -789,10 +787,18 @@ read_test(yaml_parser_t *parser, char **tables, const char *display_table, int d
 	char **table = tables;
 	while (*table) {
 		int r;
-		if (hyphenation == HYPHENATION_ON || hyphenation == HYPHENATION_BRAILLE) {
+		if (testmode == MODE_HYPHENATION || testmode == MODE_HYPHENATION_BRAILLE) {
 			r = check_hyphenation(
-					*table, word, translation, hyphenation == HYPHENATION_BRAILLE);
+					*table, word, translation, testmode == MODE_HYPHENATION_BRAILLE);
+
+		} else if (testmode == MODE_DISPLAY) {
+			r = check_display(*table, word, translation);
 		} else {
+			int direction = DIRECTION_FORWARD;
```

Comment:
Presumably `DIRECTION_FORWARD`. I think this is a bit confusing to read. Why not just put all in the `if` statement and maybe have the else handle the default? Your code seems like a micro-optimization.

Reply:
Optimization? No, didn't really think about it. I don't think it is confusing, but feel free to change.

Reply:
Tried to change but while doing that realized that it was OK as it was